### PR TITLE
Feature/#219-retrieve-profile-student-image

### PR DESCRIPTION
**Pull request by diegdar**
Get the url student image profile with the endpoint:
GET->student/{studentId}/resume/photo

### CODE REVIEW

**Review by ivanlegranbizarro — APPROVED** [review]

**Comment by JustTheDude001** [comment]
Why .env.docker empty file? I am trying to test the app but some confing parameters are missing. 
image
I think it is due the .env.docker file being empty.

File: app/Http/Requests/UpdateImageStudentRequest.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?php
+
```
Comment (jordimorillo) [inline app/Http/Requests/UpdateImageStudentRequest.php:2]: declare(strict_types=1);

**Review by jordimorillo — CHANGES_REQUESTED** [review]
Fix your branch and remove the commits not belonging to you

**Review by jordimorillo — APPROVED** [review]
Well done!

**Pull request merged**